lib/templates.ts

```diff
@@ -1454,3 +1475,43 @@ export const checkIfClosed = async (formId: string) => {
     return null;
   }
 };
+
+export const updateNotificationsSetting = async (
+  formId: string,
+  notificationsInterval: number | null
+) => {
+  const { user } = await authorization.canEditForm(formId).catch((e) => {
+    logEvent(
+      e.user.id,
+      { type: "Form", id: formId },
+      "AccessDenied",
+      "Attempted to update notifications interval for Form"
+    );
+    throw e;
+  });
+
+  if (
+    notificationsInterval !== null &&
+    (typeof notificationsInterval !== "number" || notificationsInterval < 0)
```

Comment:
This error handling should be expanded so we ensure a known value based on value i.e on of

```
OFF: null,
DAY: 1440,
WEEK: 10080,
```

I would suggest making a validation utility for that so it can be used in other places i.e. when setting marker etc...